UI/NativeApp.cpp

```diff
@@ -482,27 +495,44 @@ void NativeInit(int argc, const char *argv[], const char *savegame_dir, const ch
 	g_Config.defaultCurrentDirectory = Path("/");
 	g_Config.internalDataDirectory = Path(savegame_dir);
 
-#if defined(__ANDROID__)
-	// TODO: This needs to change in Android 12.
-	//
-	// Maybe there should be an option to use internal memory instead, but I think
-	// that for most people, using external memory (SDCard/USB Storage) makes the
-	// most sense.
-	g_Config.defaultCurrentDirectory = Path(external_dir);
+#if PPSSPP_PLATFORM(ANDROID)
+	// In Android 12 with scoped storage, due to the above, the external directory
+	// is no longer the plain root of external storage, but it's an app specific directory
+	// on external storage (g_extFilesDir).
```

Comment:
Hm, `g_extFilesDir` doesn't seem to be populated by anything?  Can we use external_dir?

-[Unknown]

Reply:
Sure it is. As stated in another comment, it's the app-private folder on external storage that we can mess around in without scoped storage tricks.

Reply:
Ah, sorry, I was looking at the diff for a new assignment.

-[Unknown]